flink-streaming-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/util/KinesisConfigUtil.java

```diff
@@ -215,4 +229,18 @@ private static void validateOptionalPositiveDoubleProperty(Properties config, St
 			}
 		}
 	}
+
+	private static void validateOptionalDateProperty(Properties config, String key, String message) {
```

Comment:
Ah, you've already checked the parsing of Date format here.
I'd say we don't really need to do the `ParseException` handling in `ShardConsumer` then, because that should never fail.